src/dynamics/video_player/player/player.js

```diff
@@ -1635,6 +1635,21 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
 
                 },
 
+                _updatePlaylistByParams: function() {
+                    const playlist = [...this.get("playlist")];
```

Comment:
Hi @Josewd, this is throwing an error when there's no playlist. Can be reproduced by running `npm run demos` and opening http://localhost:5050/static/demos/embed/test-player.html.
```suggestion
                    const playlist = [...(this.get("playlist") || [])];
```